kuma/wiki/kumascript.py

```diff
@@ -298,7 +299,11 @@ def macro_sources():
     response = requests.get(ks_macro_url)
     if response.status_code == 200:
         macros_raw = response.json()['macros']
-        return {md['name']: md['filename'] for md in macros_raw}
+        # Ensure Normal Form C used on GitHub
+        return {
+            unicodedata.normalize('NFC', md['name']):
```

Comment:
Wow, I didn't even realize there were normalization forms for unicode. Good catch and nice work @jwhitlock !